nucypher/cli/characters/alice.py

```diff
@@ -165,71 +177,87 @@ def alice(click_config,
         ALICE.log.info('Starting HTTP Character Web Controller')
         return controller.start(http_port=http_port, dry_run=dry_run)
 
+    elif action == "destroy":
+        """Delete all configuration files from the disk"""
+        if dev:
+            message = "'nucypher ursula destroy' cannot be used in --dev mode"
+            raise click.BadOptionUsage(option_name='--dev', message=message)
+        return actions.destroy_configuration(character_config=alice_config, force=force)
+
+    #
+    # Alice API
+    #
+
     elif action == "public-keys":
         response = ALICE.controller.public_keys()
         return response
 
     elif action == "create-policy":
+
+        # Validate
         if not all((bob_verifying_key, bob_encrypting_key, label)):
             raise click.BadArgumentUsage(message="--bob-verifying-key, --bob-encrypting-key, and --label are "
                                                  "required options to create a new policy.")
 
+        # Request
         create_policy_request = {
             'bob_encrypting_key': bob_encrypting_key,
             'bob_verifying_key': bob_verifying_key,
             'label': label,
             'm': m,
             'n': n,
         }
```

Comment:
The value is required for `BlockchainPolicies`. Anyway, I think we should get rid of the "create-policy" command. You cannot use the result for anything. If I'm not wrong, we already discussed this in Málaga with @vepkenez @jMyles and you.